app/boards/arm/dao/dao.keymap

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2023 ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+#define DEF 0
+#define LWR 1
+#define RSE 2
+#define ADJ 3
+
+&lt {
+    quick_tap_ms = <200>;
+};
+
+&mt {
+    quick_tap_ms = <200>;
+};
```

Comment:
I'd remove these two non-default settings unless you feel strongly about it, in which case you should use `quick-tap-ms` (with hyphens) as the non-deprecated name of the property.

Reply:
Removed from default keymap.

Reply:
The keymap the Dao originally used was ported into ZMK by me (initially for the Jian/Jorne), and I strongly feel `quick-tap-ms` is necessary here. I originally lobbied for the development of that feature specifically for use with that keymap.
The keymap is changed a bit here, but this note still applies. I will address the changes in another comment.